flux/control/controller_test.go

```diff
@@ -23,6 +24,7 @@ func TestController_Query(t *testing.T) {
 		compiler := &mock.Compiler{
 			Type: "mock",
 			CompileFn: func(ctx context.Context) (flux.Program, error) {
+				time.Sleep(time.Second)
```

Comment:
Worth adding a comment above this explaining why?

Reply:
Will do.